js/cryptocom.js

```diff
@@ -0,0 +1,1082 @@
+        }
+        if (errorCode !== '0') {
+            throw new ExchangeError (this.id + ' ' + message);
+        }
+    }
+
+    parseTicker (ticker, market = undefined) {
+        // {
+        //     "i":"CRO_BTC",
+        //     "b":0.00000890,
+        //     "k":0.00001179,
+        //     "a":0.00001042,
+        //     "t":1591770793901,
+        //     "v":14905879.59,
+        //     "h":0.00,
+        //     "l":0.00,
+        //     "c":0.00
+        // }
+        let timestamp = this.safeInteger (ticker, 't');
+        if (timestamp === undefined) {
+            timestamp = this.milliseconds ();
+        }
+        let symbol = undefined;
+        if (market !== undefined) {
+            symbol = market['symbol'];
+        }
+        let open = undefined;
+        let change = undefined;
+        let average = undefined;
+        const last = this.safeNumber (ticker, 'a');
+        const relativeChange = this.safeNumber (ticker, 'c');
+        if (relativeChange !== -1) {
+            open = last / this.sum (1, relativeChange);
+            change = last - open;
+            average = this.sum (last, open) / 2;
+        }
+        return {
+            'symbol': symbol,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'high': this.safeNumber (ticker, 'h'),
+            'low': this.safeNumber (ticker, 'l'),
+            'bid': this.safeNumber (ticker, 'b'),
+            'bidVolume': undefined,
+            'ask': this.safeNumber (ticker, 'k'),
+            'askVolume': undefined,
+            'vwap': undefined,
+            'open': open,
+            'close': last,
+            'last': last,
+            'previousClose': undefined,
+            'change': change,
+            'percentage': relativeChange * 100,
+            'average': average,
+            'baseVolume': this.safeNumber (ticker, 'v'),
+            'quoteVolume': undefined,
+            'info': ticker,
+        };
+    }
+
+    parseTrade (trade, market = undefined) {
+        //
+        // public/get-trades
+        //
+        // {"dataTime":1591710781947,"d":465533583799589409,"s":"BUY","p":2.96,"q":16.0,"t":1591710781946,"i":"ICX_CRO"},
+        //
+        // private/get-trades
+        //
+        // {
+        //     "side": "SELL",
+        //     "instrument_name": "ETH_CRO",
+        //     "fee": 0.014,
+        //     "trade_id": "367107655537806900",
+        //     "create_time": 1588777459755,
+        //     "traded_price": 7,
+        //     "traded_quantity": 1,
+        //     "fee_currency": "CRO",
+        //     "order_id": "367107623521528450"
+        // }
+        let timestamp = this.safeInteger (trade, 't');
+        if (timestamp === undefined) {
+            timestamp = this.safeInteger (trade, 'create_time');
+        }
+        let symbol = undefined;
+        if (market !== undefined) {
+            symbol = market['symbol'];
+        }
```

Comment:
here we need an else, otherwise the symbol is empty, when you execute the fetchMyTrades without a market:
```
else {
  symbol = this.safeSymbol (marketId, market, '_')
}
```

Reply:
good catch! thanks 👍